### ISSUE TITLE
`object-does-not-match-filename` complains when the name is almost right

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#460]
I see this warning:

```
org.eolang.sys.posix:17 It is expected that the "org.eolang.sys.posix" object is
coming from either "org/eolang/sys/posix.eo" or "org/eolang/sys/posix.phi" file, 
however its "program/@source" points to ".eoc/2-pull/org/eolang/sys/posix.eo" 
(object-does-not-match-filename)
```

Looks like a false alarm to me. Maybe `object-does-not-match-filename` should only check the tail of the `@source` attribute?

**Comment by yegor256** [comment]
@h1alexbel what do you think?

**Comment by h1alexbel** [comment]
@yegor256 Its a bug, we should check only relative path in the lint
